tbx/project_styleguide/templates/patterns/molecules/partner_logos/partner_logos.html

```diff
@@ -0,0 +1,7 @@
+{% load wagtailimages_tags %}
+
+{% for partner_logo in partner_logos %}
+    <div class="partners__logo-wrapper">
+        {% image partner_logo.image max-100x90 format-webp loading="lazy" class="partners__logo" %}
```

Comment:
**Issue**: This breaks the rendering of the PartnersBlock on the ServicePage, since there we expect

```django
{% image partner_logo max-100x90 format-webp loading="lazy" class="partners__logo" %}
```

Suggested fix:



```suggestion
        {% image partner_logo max-100x90 format-webp loading="lazy" class="partners__logo" %}
```

Then, 

1. defining a (cached) property on the HomePage, say, `partner_logos`:

```python
    @cached_property
    def partner_logos(self):
        if logos := self.logos.all():
            return [logo.image for logo in logos]
        return []
```

2. Updating the home_page.html template:

```diff
- {% include "patterns/molecules/partner_logos/partner_logos.html" with partner_logos=page.logos.all %}
+ <div>
+     {% include "patterns/molecules/partner_logos/partner_logos.html" with partner_logos=page.partner_logos %}
+ </div>
```

3. Updating the pattern library with the changes